### ISSUE TITLE
Big problems about babel/i18n

### ISSUE THREAD

**Issue by xiaohanyu** [quokkaproject/quokka#109]
Hi, I've found that even I set

```
BABEL_LANGUAGES = ['zh', 'en']
BABEL_DEFAULT_LOCALE = 'zh'
```

in `local_settings.py`, it still didn't work for me. 

If if works, by default the flask-admin will show some text in zh locale. 

I've found the problem may exist in 

```
    babel = app.extensions.get('babel')
    if babel:
        try:
            @babel.localeselector
            def get_locale():
                override = request.args.get('lang')

                if override:
                    session['lang'] = override

                return session.get('lang', 'en')
            admin.locale_selector(get_locale)
        except:
            pass  # Exception: Can not add locale_selector second time.
```

since this code block always returns 'en' as the default locale. But I'm not sure.

**Comment by ottk3** [comment]
My solution for that part in [**init**.py](https://github.com/pythonhub/quokka/blob/4a5bdd8f17eaadf31966efb00541d4f78d4c949b/quokka/core/admin/__init__.py#L67). It uses `BABEL_DEFAULT_LOCALE` if set, so set `BABEL_DEFAULT_LOCALE = Null` if you want to use matching for the best language of given `BABEL_LANGUAGES`.

``` python
    babel = app.extensions.get('babel')
    if babel:
        try:
            @babel.localeselector
            def get_locale():
                # use default language if set
                if app.config.get('BABEL_DEFAULT_LOCALE'):
                    session['lang'] = app.config.get('BABEL_DEFAULT_LOCALE')
                else:
                    # get best matching language
                    if app.config.get('BABEL_LANGUAGES'):
                        session['lang'] = request.accept_languages.best_match(app.config.get('BABEL_LANGUAGES'))

                return session.get('lang', 'en')

            admin.locale_selector(get_locale)
        except:
            pass  # Exception: Can not add locale_selector second time.

```

If this is how it's intended, the `BABEL_DEFAULT_LOCALE` should be renamed to `FORCE_LOCALE`, otherwise i would kill the complete `BABEL_DEFAULT_LOCALE` part, because you can just only add languages you like to `BABEL_LANGUAGES`.

What do you guys think about it?

**Comment by rochacbruno** [comment]
I guess it was fixed by #156
